src/domain/moit/components/TimeZone.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from 'react';
+import styled from '@emotion/styled';
+
+interface Props {
+  currentCursor: TimeZoneCursor;
+  startTime: TimeParams;
+  endTime: TimeParams;
+  onTimeZoneClick: (type: TimeZoneCursor) => void;
+}
+
+const TimeZone: FC<Props> = ({
+  currentCursor,
+  startTime,
+  endTime,
+  onTimeZoneClick,
+}) => {
+  return (
+    <Container>
+      <Content
+        isCurrentCursor={currentCursor === 'start'}
+        onClick={() => onTimeZoneClick('start')}
+      >
+        <p>시작</p>
+        <span>{`${startTime.hour}시 `}</span>
+        <span>{`${startTime.minute}분 `}</span>
+      </Content>
+      <Content
+        isCurrentCursor={currentCursor === 'end'}
+        onClick={() => onTimeZoneClick('end')}
+      >
+        <p>종료</p>
+        <span>{`${endTime.hour}시 `}</span>
+        <span>{`${endTime.minute}분 `}</span>
+      </Content>
+    </Container>
+  );
+};
+
+export default TimeZone;
+
+const Container = styled.section`
+  display: flex;
+`;
+
+const Content = styled.div<{ isCurrentCursor: boolean }>`
+  width: 50%;
+  font-weight: 600;
+
+  p {
+    font-size: 16px;
+    line-height: 23px;
+    color: ${({ isCurrentCursor, theme }) =>
+      isCurrentCursor ? theme.palette.blue500 : theme.palette.gray300};
```

Comment:
P2
```suggestion
    color: ${({ theme, isCurrentCursor }) =>
      theme.palette[isCurrentCursor ? 'blue500' : 'gray300']};
```
`SelectScroller.option.tsx`에서 한 스타일링이랑 일관성 있게 가져가는건 어때? 반대여도 좋고 편한대로!